### DISCUSSION TITLE
Git .mat and .slx files not downloaded in MATLAB

### DISCUSSION THREAD

**Discussion by grisamgit** [DLR-VSDC/IEEE-MVC-2023#19] [Q&A] [answered]
Hi everyone.
I retrieved the whole folder from Git in MATLAB (2022b).
However, .slx and .mat files are not downloaded. They are pointers to a file.
I also installed LFS support, but nothing changed.
I had to manually download them to make it work.

Actually, it still does not work properly, as there is an issue with an 'access violation' of some .mex file.

Thanks in advance for your assistance.

Samuele

**Comment by JBRDLR** [comment, marked as answer]
Hi,
we suggest to install: Git 2.39.1, Git LFS 3.3.0 and TortoiseGit 2.10.0.2 in Windows.
Please check your installation in the command line (cmd):
`git lfs --version`
Please make sure that in the path, where you clone the repository are **no special characters in the pathname**.
C.f. https://github.com/DLR-VSDC/IEEE-MVC-2023/discussions/7#discussioncomment-4506723

Moreover, we strongly recommend to use MATLAB 2021b as described in the installation routine.
https://github.com/DLR-VSDC/IEEE-MVC-2023/blob/main/Installation.md

Best 
Jonathan
